docs/docs/introduction.md

```diff
@@ -0,0 +1,104 @@
+Introduction
+============
+
+Actinia is a REST service to process geographical data that can be
+managed by the GRASS GIS software system. The software is designed to
+expose a GRASS GIS database and many GRASS GIS[^1] processing tool as
+REST service[^2]. Hence, access to GRASS resources like raster maps,
+space-time raster datasets, processing and analysis modules are
+available via URL. In addition Actinia allows the processing of cloud
+based data, for example all Landsat 4-8 scenes as well as all
+Sentinel-2A scenes in an ephemeral databases. The computational results
+of ephemeral processing are available via object storage as GeoTIFF
+files.
+
+The actinia service consists of the *actinia core* that provides the
+basic but sophisticated processing service and *actinia plugins* that
+provide problem specific services like Sentinel-2 and Landsat NDVI
+computation, spatio-temporal statistical analysis and many more.
+
+To use actinia the user must have an understanding of the GRASS GIS
+concept[^3] of location, mapsets, raster maps, space-time datasets and
+modules. The URLs that provide access to the GRASS database reflect
+these concepts. Hence, the location, the mapset, the required raster map
+are part of the URL to access the service.
+
+What is REST?
+-------------
+
+The Representational state transfer (REST)[^4] is an architectural style
+based on HTTP[^5] that uses GET[^6], DELETE[^7], POST[^8] and PUT[^9]
+methods to manipulate and receive resource with stateless operations.
+
+While GET requests can be send easily from a browser, POST, PUT or
+DELETE request can not. To access the full potential of actinia you will
+need a HTTP client, that talks all HTTP communication methods.
+
+Examples
+--------
+
+**Data management**
+
+List all locations that are available in the actinia persistent database:
+
+```bash
+ curl -X GET "https://actinia.mundialis.de/api/v1/locations" -H "authorization: Basic ..."
```

Comment:
I dont' understand what "Basic ..." for authorization means. Is this "user" or also "guest" user-role? Maybe one example showing how to correctly put an authorization here would also help.